Issues #1614/Arrows missing in WP 4.8

endifmedia opened the pull request:
reference https://github.com/Strategy11/formidable/issues/1614

* added recommendations from issue tracking (above)
* changed **widget-title** h4 to h3
* updated css to fix down arrow not rotating when dropdown is open
* fixed vertical alignment on action icons left of title

stephywells commented:
Thank you so much! Great solution. It looks great for the field options box in the current WordPress version. I checked the field options and form actions boxes in WP 4.6 too, and it looks like it'll need a little extra love. Here are a few screenshots of what I'm seeing.

WP 4.6:
<img width="752" alt="4 6-field" src="https://user-images.githubusercontent.com/1116876/31639814-0251a104-b298-11e7-9302-ad94cd818940.png">
<img width="613" alt="4 6-action" src="https://user-images.githubusercontent.com/1116876/31639817-05736958-b298-11e7-9805-a1fe605487e9.png">

Also, the arrow isn't switching on the form actions in 4.8. And is it possible to give that left icon a bit more margin before the action title?
<img width="616" alt="4 8-action" src="https://user-images.githubusercontent.com/1116876/31639825-15870b1a-b298-11e7-9ffb-6b5126344e66.png">

endifmedia commented:
Good catch. Added a bit of css for backwards compatibility and fixed an issue with the dropdown arrow not rotating correctly when a new action is added. Tested back to 4.0, looks good.

stephywells commented on css/font_icons.css:
```diff
@@ -210,6 +247,8 @@ a.frm_icon_font:hover{
 .frm_email_icon:before {
 	content: "\e626";
+	vertical-align: text-top;
+	margin-right: 5px;
```
Since this class can be used in other locations, and on the front-end, we'll need to make this addition in the frm_admin.js file.

endifmedia commented:
good call. css moved over to frm_admin.css

stephywells commented:
Thank you! This is all merged.
